dashboard/test/controllers/followers_controller_test.rb

```diff
@@ -33,6 +33,14 @@ class FollowersControllerTest < ActionController::TestCase
     assert_redirected_to controller: 'sections', action: 'show', id: @word_section.code
   end
 
+  test "section code with preceding and trailing whitespaces should redirect to login" do
+    get :student_user_new, params: {section_code: @word_section.code}
+    assert_redirected_to controller: 'sections', action: 'show', id: @word_section.code
+
+    get :student_user_new, params: {section_code: @picture_section.code}
+    assert_redirected_to controller: 'sections', action: 'show', id: @picture_section.code
+  end
```

Comment:
Hi @indhu-gunda , thank you for your hard work on this feature, and thank you for adding tests! It looks like the description of the test is correct, but in the body of the test, I do not see a section code with leading or trailing whitespace. Could you please update the test to add the whitespace to the section code?

Also, please let us know if you are having any trouble running the tests locally.

Reply:
Also let me know if you want to pair on creating this test. Happy to do that!

Reply:
My mistake and I will change that. I was trying to get the testing working locally and forgot the whole point of the test.

Reply:
Should be fixed now. Sorry about that!

Reply:
the new test code looks good! There seems to be a problem with the Drone test run, though. I'm not sure if this will do the trick, but can you please try merging our latest staging branch into your own, and then push your branch again? Thanks!

Reply:
Hi @indhu-gunda , apologies, but never mind pushing your branch again. @dmcavoy says she will make drone run on it for you since it needs to be done by someone on our team.